CONTRIBUTING.md

```diff
@@ -11,6 +11,161 @@ requirements and recommendations.
 Before you can contribute, you will need to sign the [Contributor License
 Agreement](https://identity.linuxfoundation.org/projects/cncf).
 
-## Under Construction
+## Find a Buddy and Get Started Quickly
 
-TBD
+If you are looking for someone to help you find a starting point and be a
+resource for your first contribution, join our Slack channel and find a buddy!
+
+1. Create your [CNCF Slack account](http://slack.cncf.io/) and join the
+   [otel-community-demo](https://app.slack.com/client/T08PSQ7BQ/C03B4CWV4DA) channel.
+2. Post in the room with an introduction to yourself, what area you are
+   interested in (check issues marked with [help
+   wanted](https://github.com/open-telemetry/opentelemetry-demo-webstore/labels/help%20wanted)),
+   and say you are looking for a buddy. We will match you with someone who has
+   experience in that area.
+
+Your OpenTelemetry buddy is your resource to talk to directly on all aspects of
+contributing to OpenTelemetry: providing context, reviewing PRs, and helping
+those get merged. Buddies will not be available 24/7, but is committed to
+responding during their normal contribution hours.
+
+## Development Environment
+
+You can contribute to this project from a Windows, macOS or Linux machine. The
+first step to contributing is ensuring you can run the demo successfully from
+your local machine.
+
+On all platforms, the minimum requirements are:
+
+- Docker
+- [Docker Compose](https://docs.docker.com/compose/install/#install-compose) v2.0.0+
+
+### Clone Repo
+
+- Clone the Webstore Demo repository:
+
+```shell
+git clone https://github.com/open-telemetry/opentelemetry-demo-webstore.git
+```
+
+### Open Folder
+
+- Navigate to the cloned folder:
+
+```shell
+cd opentelemetry-demo-webstore/
+```
+
+### Gradle Update [Windows Only]
+
+- Navigate to the Java Ad Service folder to install and update Gradle:
+
+```shell
+cd .\src\adservice\
```

Comment:
Consider using forward slashes.

Reply:
What's the difference?

Reply:
Forward slashes are native to Linux/Mac/Unix. Backward slashes are native to Windows.

When it comes to cross-platform documentation, either use `/` by default, or have separate instructions for different platforms.

Reply:
The ad service gradlew piece is a windows only issue. It's not meant to be a cross platform step. If it doesn't break windows users I can update it

Reply:
Ah I see, never mind then (`/` won't break modern Windows though).